OCI MFA concept doc (#5657)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/concepts/mfa/index.md

---

#### Learning outcomes

* Learn the differences between authenticators, factors, and methods.
* Understand how MFA works in Okta.
* Understand how MFA works with policies.

---

## What is multifactor authentication

MFA requires users to supply two or more types of evidence that prove they are who they claim to be:

* **Something they know (knowledge-based factor, like a password or PIN):** Authenticators that satisfy a knowledge factor rely on information that's only known to the user.
* **Something they have (possession-based factor, like a smartphone or hardware token):** Authenticators that satisfy a possession factor prove that the user has a specific device in their control. For example, a user who enters a code from an authenticator app or approves a push notification demonstrates their possession of that device.
* **Something they are (biometric-based factor, like a fingerprint or facial recognition):** Authenticators that satisfy a biometric factor verify a user's identity by scanning unique traits such as a fingerprint or using facial recognition.

> Some authenticators can satisfy more than one factor type. For example, Okta Verify can be used as a possession factor and as a biometric factor.

Reviewer: ```suggestion
Some authenticators can satisfy more than one factor type. For example, you can use Okta Verify as a possession factor and as a biometric factor.
```

## Key terms with MFA

* **Assurance**: The level of confidence that a user is who they claim to be, based on the authentication methods used.
* **Factor**: The category of authentication (knowledge, possession, biometric).
   > **Note:** In Classic Engine, "factor" is used interchangeably with "authenticator." In Identity Engine, a factor refers to the category of authentication (knowledge, possession, biometric), while an authenticator is a specific method or device. See [Understand the terminology in the User Factors API](#understand-the-terminology-in-the-user-factors-api).
* **Authenticator**: A method or device that a user possesses and controls and that they use to verify their identity. Authenticators can be used to satisfy different factor requirements. For example, Okta supports authenticators like Okta Verify, Smart Cards, and YubiKey One-Time Passcodes (OTPs), among others.
* **Authenticator method**: The security method used by an authenticator. Typically the authenticator method describes the protocol that the authenticator uses. See [Method characteristics](https://help.okta.com/okta_help.htm?type=oie&id=ext-about-authenticators).

> * **Authenticator enrollment**: The specific instance of an authenticator that a user has enrolled and that's linked to them and their account. It refers to the specific configuration of the authenticator and the user who's enrolled it.

Reviewer 2: ```suggestion
* **Authenticator enrollment**: The specific instance of an authenticator that a user has enrolled and that's linked to them and their account. It refers to the specific configuration of the authenticator and the user who enrolled it.
```

For more conceptual information about MFA, see [What Is Multi-Factor Authentication (MFA)?](https://www.okta.com/blog/2021/08/multi-factor-authentication-mfa/)

### Understand authenticator terminology with Okta APIs

The terminology of authenticators in Okta's APIs is similar to the terminology that's used in the Admin Console. However, there are some slight differences in how they're represented in the APIs.

| Key term                  | Description  |
|---------------------------|--------------|
| [Authenticator key](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Authenticator/#tag/Authenticator/operation/listAuthenticators!c=200&path=0/key&t=response) | The authenticator `key` parameter is the name of the authenticator. |
| [Authenticator type](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Authenticator/#tag/Authenticator/operation/listAuthenticators!c=200&path=0/type&t=response) | The authenticator `type` parameter is a category for authenticators. There are various authenticator types and the parameter is used to group authenticators that rely on similar technologies. |

> | [Method type](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Authenticator/#tag/Authenticator/operation/listAuthenticatorMethods!c=200&path=13/type&t=response) | The specific security method that an authenticator can use for verification. In the Authenticators API, the method `type` parameter refers to the specific security methods of an authenticator. It doesn't refer to any of the method characteristics, such as user presence or device-bound. |

Reviewer: ```suggestion
| [Method type](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Authenticator/#tag/Authenticator/operation/listAuthenticatorMethods!c=200&path=13/type&t=response) | The specific security method that an authenticator can use for verification. In the [Authenticators API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Authenticator/), the method `type` parameter refers to the specific security methods of an authenticator. It doesn't refer to any of the method characteristics, such as user presence or device-bound. |
```

> | [Authenticator enrollment](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/UserAuthenticatorEnrollments/) | The instance of an authenticator that is registered to a specific user. |

Reviewer: ```suggestion
| [Authenticator enrollment](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/UserAuthenticatorEnrollments/) | The instance of an authenticator that's registered to a specific user. |
```

To learn how to configure authenticator enrollment policies in the Admin Console, see [Authenticator enrollment policies](https://help.okta.com/okta_help.htm?type=oie&id=ext-create-mfa-policy).

You can also configure them by using the [Policy API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Policy/).

## About MFA and authentication policies

Authentication policies determine the extra levels of authentication that are performed before a user can access an app. Admins can use these policies to specify the required level of assurance for different apps, user groups, or access scenarios. For example, you can set up an authentication policy where the required authenticators change based on the user's network location and device platform.

To learn how to configure authenticator enrollment policies in the Admin Console, see [Add an authentication policy rule](https://help.okta.com/okta_help.htm?type=oie&id=ext-create-auth-policy).

You can also configure them by using the [Policy API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Policy/).

## Get started with MFA in your org

> * [Learn about authentication in the Admin Console](https://help.okta.com/oie/en-us/content/topics/security/security_at_okta.htm)

Reviewer: needs an alias link

> * [Configure authenticators in the Admin Console](https://help.okta.com/oie/en-us/content/topics/identity-engine/authenticators/about-authenticators.htm)

Reviewer: needs an alias link
